cfg/none.cfg

```diff
@@ -3,10 +3,11 @@ SYMBOLS {
 }
 MEMORY {
     ZP:   file = "", define = yes, start = $0000, size = $0001F;
-    MAIN: file = %O,               start = %S,    size = $10000 - __STACKSIZE__;
+    MAIN: file = %O, define = yes, start = %S,    size = $10000 - __STACKSIZE__;
```

Comment:
Sorry, I didn't notice right away that the none.cfg had (another) bug before you started your pull request. Anyhow, you need to subtract the start address from the size to have and "end address". See https://github.com/cc65/cc65/blob/master/cfg/apple2.cfg#L16 as an example how this is meant to be done (incl. using a symbol).